### DISCUSSION TITLE
build mxnet with cuda11.6+cudnn8.x+vs2022+rtx3090+win10

### DISCUSSION THREAD

**Discussion by overcash597** [apache/incubator-mxnet#20883] [Q&A]
hi ,I am a user of mxnet ,I used mxnet-cu92 +anaconda 3.x for some study of deep learning. now I get a RTX309,it can only run in cuda 11.X,so it only one way to rebuild source code by myself, I build the source code under the instruction of mxnet . after a long compile time past,it raised 7 error to me :
link 2019 unresolved external symbol __imp__localtime64_s referenced in mz_zip_writer_add_mem_ex_v2  project mxnet_86
link 2019 unresolved external symbol __imp__mktime64 referenced in mz_zip_file_stat_internal                  project mxnet_86
link 2019 unresolved external symbol __imp__time64 referenced in mz_zip_writer_add_mem_ex_v2            project mxnet_86
link 2019 unresolved external symbol __imp__foreopen_s referenced in mz_zip_writer_init_from_reader_v2    project mxnet_86
......
so libmxnet.lib can not generate....
I do not know how to fix it, Any suggestion will be appriciated!

**Comment by chinakook** [comment]
Your mxnet version may be very old, so it cannot support vs2019 and vs2022.
For a compatible version, Install the vs2019's C++ toolset from the vs installer. Use cmake -T to specify the vs2019's C++ toolset version and then build.

**Reply by overcash597** [reply]
hi,the source of mxnet I git from ubcubator-mxnet that 's older one? (where  can get nearest one?)thank you for reply my request

**Comment by GeoffNN** [comment]
Maybe this is a stale issue -- the latest version of mxnet is compatible with cuda 11.2. Use `pip install mxnet-cu112` with cuda 11.2 installed.
